package.json

```diff
@@ -54,6 +53,8 @@
     "eslint": "^6.8.0",
     "mocha": "10",
     "node-gyp": "^9.4.0",
+    "node-addon-api": "^7.0.0",
+    "node-api-headers": "^1.1.0",
```

Comment:
`node-api-headers` is needed for users who didn't install full nodejs headers when installing nodejs. I have removed this because node-gyp seems to install it automatically.